Source/ACE/Entity/Container.cs

```diff
@@ -26,48 +29,111 @@ public Container(ObjectType type, ObjectGuid guid, string name, ushort weenieCla
         public Container(AceObject baseObject)
             : base(baseObject) { }
 
+        public Container(ObjectGuid guid, AceObject baseAceObject)
```

Comment:
On looking at this more - the issue is I never finished moving physics data and model data into world object.   If I finish the move, then those properties are expressed using the aceobject as it's backing object.   Good news is - it is simple to finish the move.  The bad news is, I will touch or change 50% of the files to do it.   I think that needs to be a clean - that is the only thing I do PR.   Let me know what you want me to do.     physicsData.cSetup goes away and this shows up in world object for example.

public uint? CSetup
        {
            get { return AceObject.ModelTableId; }
            set { AceObject.ModelTableId = value; }
        }